### PR TITLE
Grill_capstone-1 Project

### PR DESCRIPTION
**Pull request by sarazaToussaint**
In this project we:

- Add implementation of the mobile and the desktop version for  the home page for the Grill restaurant
- Add implementation of the mobile and the desktop version for  the about page for the Grill restaurant
- Create an object and the function to make the featured speaker's section dynamic when the website is loads

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,56 @@
...
+- npx webhint .
+- npx stylelint "**/*.{css,scss}"
 
-### Deployment
-
-
-
-## Authors
+### Documentation
+[Loom Video Link] (https://www.loom.com/share/4d2140998d4e4804936ec8408637fe35)
```
Comment (nicupop729) [inline README.md]: Congratulations for describing the project and showing a demo of the project features (different pages and different screen sizes)

But the project requirements state that the video should also include a highlight of some interesting piece of code or something you built that you are very proud of.

- [x] Kindly, rerecord the video and besides the presentation of your project and the description of it, including a highlight of some interesting piece of code or something you built that you are very proud of.

File: README.md [hunk 2]
```diff
@@ -1,73 +1,56 @@
...
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
```
Comment (nicupop729) [inline README.md:70]: We always need to give appropriate credit to those who inspired us.

- [x] Please, reintroduce the Acknowledgements section in the README.md file and give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Best Grill</title>
+     
+      <link rel="stylesheet" href="main.css">
+  </head>
+       <body>
+             <header>
+                    <div class="container">
+                        <div>
+                          <h2 class="logo"><a href="index.html"><img src="./components/resto.png" alt=""></a></h2>
```
Comment (nicupop729) [inline index.html:32]: - [x] In order to respect the given design that we need to follow, kindly, introduce to your project, for both Home and About pages, the top nav bar;
Given design:
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+                          <img src="./components/bg2.jpg" alt="service4 image">
+                           <div>
+                               <h4>Salade</h4>
+                               <p>
+                                  Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                               </p>
+                           </div>
+                      </div> 
+                    </div>
+                 </aside>
```
Comment (nicupop729) [inline index.html]: - [x] In order to respect the design, kindly, implement another card in the Signature dishes section;

Your design:
image

Given design:
image

Comment (nicupop729) [reply index.html]: - [x] Also, in order to respect the design, please add a anchor tag/button under the cards:

Given design for desktop:
image

Given design for mobile:
image

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi  @sarazaToussaint, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Some highlights for you
* Correct GitHub flow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 
* Clean codebase ✔️ 
* Linters are passing ✔️

### Required Changes ♻️

- [ ] _Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (sarazaToussaint) [reply index.html]: Hello @nicupop729, 
Thank you for your feedback, I made all the changes you requested and I think my PR is ready to get approved.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+                            <li class="campain"><a href="">CC Campain</a></li>
+                          </ul>
+                        </nav>
+                    </div> 
+                    <div>
+                      
+                    </div>
+                    <div class="main">
+                      <h2>"Hello Sharing World"</h2>
+                      <h1>WELCOM TO THE BEST GRILL</h1>
```
Comment (Meltrust) [inline index.html]: - [x] OPTIONAL: There is a typo here!  "WELCOM" should be "WELCOME".  Please, I recommend you fix this.

**Review by Meltrust — APPROVED** [review]
Hi @sarazaToussaint,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing all the changes requested by the previous reviewer, including improving your readme, the styling, adding missing elements of the page, and improving your video to include a piece of code you are proud of! 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (sarazaToussaint) [reply index.html]: Hi @Meltrust, thank you for approving my PR and mentioning this typo, I fixed it

**Pull request merged**